src/float_parser.ts

```diff
@@ -0,0 +1,39 @@
+import BN from "bn.js";
+
+const buf = new ArrayBuffer(8);
+const view = new DataView(buf);
+
+export function numberToFraction(number: number) {
+  // retrieve mantissa, exponent, and sign from bit representation of number
```

Comment:
I would move this comment over the function and have it as a doc comment, that way the language service can give you awesome intellisense.

See example format here: https://github.com/gnosis/dex-contracts/blob/master/test/resources/math.ts#L6-L10